### PR TITLE
added redis calls LLEN, LINDEX, LPOP and LRANGE

### PR DESCRIPTION
**Pull request by frequenc1**

### CODE REVIEW

**Comment by tarjei** [comment]
Hi, I would love to see this PR move forward, anything I can do?

**Comment by omansour** [comment]
@tarjei you can take the code produced by @frequenc1 on another PR 

from the master.
just not change the identation, and I think it will be mergeable

**Comment by frequenc1** [comment]
I fixed indentation. Had project set to code sweep setting used at work!

**Comment by omansour** [comment]
sorry @frequenc1 but the PR cannot still be merged
is it possible to resolve the conflicts with the master ?

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "m6web/redis-mock",
+    "name": "viverae/redis-mock",
```
Comment (mikaelrandy) [inline composer.json]: :-1: 
Will not be merge as long as this modification is still here

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 2]
```diff
@@ -795,4 +987,41 @@ protected function deleteOnTtlExpired($key)
...
+     * @param boolean $serialize_set
+     */
+    public function setUseSerializer($use_serializer)
+    {
+        $this->use_serializer = $use_serializer;
+    }
+
+    protected function isSerializedValue($value)
+    {
+        $data = @unserialize($value);
```
Comment (mikaelrandy) [inline src/M6Web/Component/RedisMock/RedisMock.php]: Why need to hide errors ?

**Comment by mikaelrandy** [comment]
I think this PR is too old and that the master has too evolved since.

Please check #42 insead

**Comment by frequenc1** [comment]
Wish I had time to do the merges. Wondering if moving the command types to traits would be an except able way to modulize that class?

**Comment by mikaelrandy** [comment]
As #42 is now merged, i close this PR.

Thanks @frequenc1 for the initial work.

@tarjei, you can now use this new functions ;)

**Comment by tarjei** [comment]
Thanks a bundle!  I'll look into adding lpush and blpop sometime in the
future
5. okt. 2015 2.53 p.m. skrev "Mikael RANDY" <notifications@github.com>:

**Pull request closed without merging**